README.rst

```diff
@@ -534,6 +585,13 @@ Polyspace
   `column title <Exporting Polyspace Results_>`_ in lowercase as the variable name.
   The default template is ``Polyspace: $check``.
 
+Coverity
+  Named groups of the regular expression can also be used, next to environment variables.
+  The capturing groups are: `path`, `line`, `column`, `curr`, `max`, `checker` and `classification`.
+  `curr` and `max` represents respectively the current warning and the maximum amount of warnings with the same CID.
+  These are always the same if you take the note of `Parse for Coverity Defects`_ into account.
```

Comment:
```suggestion
  Named groups of the regular expression can be used as variables.
  Useful names are: `checker` and `classification`.
```
The path, line and column info are already stored as separate fields for the CQ report. I don't see a use case for curr and max, so I suggest leaving it out of the documentation. What do you think?

Reply:
Yes, that's clearer